src/factory.js

```diff
@@ -2,7 +2,7 @@
  * @license
  * Blockly Demos: Block Factory
  *
- * Copyright 2016 Google Inc.
+ * Copyright 2017 Google Inc.
```

Comment:
ditto about non-new files. Feel free to ignore me if these are new files based on the existing files and not just a moved version of it.